libkbfs/config_local.go

```diff
@@ -208,6 +209,15 @@ func MakeLocalUsers(users []libkb.NormalizedUsername) []LocalUser {
 	return localUsers
 }
 
+func getDefaultCleanBlockCacheCapacity() uint64 {
+	vmstat, err := mem.VirtualMemory()
+	if err == nil {
+		// By default,  we try to cap at 1/8 of available memory
+		return vmstat.Total / 8
```

Comment:
I'm not sure we should get too greedy here.  For example, on my desktop, this would take 4 GB for KBFS.  Maybe we should cap the memory at `MaxBlockSizeBytesDefault * 1024`?  I don't feel too strongly about it though, maybe it's ok to use that much if we have lots of memory.